content/learning-paths/cross-platform/integer-vs-floats/introduction-integer-float-types.md

```diff
@@ -118,42 +116,42 @@ x = 99999.98437500000000, 0x47c34ffe
 x = 99999.99218750000000, 0x47c34fff
 ```
 
-Notice that the initial value of `x` is not the one that you assigned, but closest float value. The hex value `0x47c34ffd` corresponds to `99999.9765625`. 
+Notice that the initial value of `x` is not the one that you assigned, but the closest float value. The hex value `0x47c34ffd` corresponds to `99999.9765625`. 
 
 
-Here, the difference is in the 2nd decimal digit. After the 7th decimal digit, all the rest are zeroes! That is because 32-bit floating point numbers offer up to 7 decimal digits of precision only.
+Here, the difference is in the 2nd decimal digit. After the 7th decimal digit, all the rest are zeroes! This is because 32-bit floating point numbers offer up to 7 decimal digits of precision only.
 
-A similar point can be made for double, but because the precision is twice that of 32-bit floats, you can get 14 decimal digits of precision.
+A similar point can be made for doubles but because the precision is twice that of 32-bit floats, you get 14 decimal digits of precision.
 
 {{% notice Summary %}}
 You are not dealing with real numbers, but approximate representations of real numbers using limited precision data types. 
 
 If you assign a real number value to a float data type, the result might not be the same real number you assigned.
 {{% /notice %}}
 
-If your calculations require full precision, then you should be looking at arbitrary precision libraries such as [The GNU Multiple Precision Arithmetic Library](https://gmplib.org/). 
+If your calculations require complete precision, then you should be looking at arbitrary precision libraries such as [The GNU Multiple Precision Arithmetic Library](https://gmplib.org/). 
```

Comment:
either full precision is used or absolute precision as terminology in such contexts.

Reply:
Thanks. Will revert back to full